src/riak_core_ring_manager.erl

```diff
@@ -721,5 +732,14 @@ do_write_ringfile_test() ->
     ?assertMatch({error,_}, do_write_ringfile(GenR(ring_perms), ?TEST_RINGFILE)),
     ok = file:change_mode(?TEST_RINGDIR, 8#00755).
 
--endif.
+is_stable_ring_test() ->
+    % This test is efferently coupled to the PROMOTE_TIMEOUT macro value
```

Comment:
Yes. Its a 'note to self' that changes to PROMOTE_TIMEOUT will change the boundaries of the chosen values in the test. From the point of view of a user this is surprising unless you read the code... You know, the comment works better without reference to efference at all anyway, so I'll remove...